packages/storybook/stories/QuickStarts/QuickstartStatefulCall.stories.mdx

```diff
@@ -0,0 +1,173 @@
+
+`App.tsx`
+
+<Source code={CallAppStateful} />
+
+## Add Providers
+
+Next we will add four providers that will encapsulate our UI components.
+
+1. The `FluentThemeProvider` which provides visual theme to the underlying components.
+2. The `CallClientProvider` which provides state from the `statefulCallClient` to the components.
+   For this provider, we will pass the `statefulCallClient` as a prop to the provider.
+3. The `CallAgentProvider` which provides state from the `CallAgent` to the components.
+   For this provider, we will pass the `CallAgent` as a prop to the provider.
+4. we will add the 'CallProvider' which provides the `Call` objects state down to the components.
+   For this provider, we will pass the `Call` as a prop to the provider.
+
+`App.tsx`
+
+<Source code={CallAppStatefulProviders} />
+
+## Compose Call Components
+
+Now that we have our structure ready, we will now create our call experience using UI Components.
+For Call, we will use the [VideoGallery](./?path=/story/ui-components-videogallery--video-gallery) and [ControlBar](./?path=/story/ui-components-controlbar--control-bar) components.
+To start add the code below to a new file `CallComponentsStateful.tsx`.
+For more information on getting started with UI Components follow the [quickstart](./?path=/story/quickstarts-uicomponents--page)
+Here we will use the `usePropsFor` hook to connect our components to the stateful clients.
+The `usePropsFor` method takes in the component for which we want props generated, in this case `VideoGallery` and `ControlBar`.
+
+`CallComponentsStateful.tsx`
+
+<Source code={CallComponentsStateful} />
+
+Similarly to how we used `usePropsFor` with chat UI Components, it can be used for calling UI Components.
+For more information on [usePropsFor](./?path=/story/statefulclient-reacthooks-usepropsfor--page).
+
+## Add Components to App
+
+Now that we have configured our providers and the call components we can put them together.
+
+`App.tsx`
+
+<Source code={CallAppStatefulComplete} />
+
+## Run Quickstart
+
+At this point your app should have two UI Components hooked up to
+To run the code above use the command:
+
+```console
+
+npm run start
+
+```
+
+![End state of stateful calling quickstart]("images/StatefulEnd.png")
+
+## Troubleshooting
+
+See the [troubleshooting page](./?path=/story/troubleshooting--page) for some common problems and recommended solutions.
+
+## Clean Up Resources
+
+If you want to clean up and remove a Communication Services subscription, you can delete the resource or resource group.
+Deleting the resource group also deletes any other resources associated with it.
+Learn more about [cleaning up resources](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp#clean-up-resources).
+
+## Next Steps
+
+[Try UI Library Composite Components](./?path=/docs/quickstarts-composites--page)
```

Comment:
We should avoid the confusing term "Composite Components". Just call them "Composites" everywhere.

Also, it seems wrong that the next step after stateful quickstart is composites. We would like folks to have tried the composites first.